e2e/package.json

```diff
@@ -13,7 +13,7 @@
     "test": "cross-env WORKBENCH_ENV=test yarn _test --maxWorkers=5",
     "test:debugTest": "cross-env PUPPETEER_DEBUG=true yarn _test:debugTest",
     "test:debug": "cross-env PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false yarn _test --detectOpenHandles --maxWorkers=1",
-    "test:ci": "yarn _test --ci --maxWorkers=3 --forceExit",
+    "test:ci": "node -r dotenv/config . && yarn jest --no-color --ci --maxWorkers=3 --forceExit",
```

Comment:
This change should not be necessary. If the user tokens are already there, it will check in Ruby and immediately exit (does not touch gradle, Java).

I would probably have a slight preference for keeping these commands unified (previous approach) rather than forcing that CI prebuilds impersonation tokens at this level.

Reply:
I thought it was a good idea to make separate cmd. Do you prefer no change at all?

Reply:
I think it's fine if there are specific flags you need to run on Circle, but probably I would keep it as similar as possible to the normal local test command (using a common internal command as previously seems good).

Not sure what you mean by no change at all though; here I'm just referring to the changes made in this PR to this line.